src/Data/Mapping/Masa.Contrib.Data.Mapping.Mapster/Internal/ConcurrentDictionaryExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Data.Mapping.Mapster.Internal;
+
+internal static class ConcurrentDictionaryExtensions
```

Comment:
I don't think you need to copy and paste this lazy method all over the place, it's going to be used a lot.